pypit/ararc.py

```diff
@@ -211,11 +211,11 @@ def setup_param(slf, sc, det, fitsdict):
         else:
             msgs.error('Not ready for this disperser {:s}!'.format(disperser))
     elif sname=='lris_blue':
-        lamps = ['ZnI','CdI','HgI']
+        lamps = ['NeI', 'ArI', 'CdI', 'KrI', 'XeI', 'ZnI','CdI','HgI']
```

Comment:
I don't think this is s a good idea as the default for LRISb.
I think we'll need to allow an optional list.  @AlexaVillaume , do
you want to try to negotiate arparse? :)    Probably not.

I suggest we meet on Friday to do this together.